docs/tasks/manage-hugepages/scheduling-hugepages.md

```diff
@@ -0,0 +1,78 @@
+---
+approvers:
+- derekwaynecarr
+title: Scheduling HugePages
+---
+
+{% capture overview %}
+
+Kubernetes includes **alpha** support for managing huge pages spread across
+nodes. This page describes how users can consume huge pages and the current
+limitations.
+
+{% endcapture %}
+
+{% capture prerequisites %}
+
+1. Kubernetes nodes must pre-allocate huge pages in order for the node to
+   discover them.  A node may only pre-allocate huge pages for a single size.
+1. A special **alpha** feature gate `HugePages` has to be set to true across the
+   system: `--feature-gates="HugePages=true"`.
+
+The nodes will automatically discover and expose all huge page resources as a
+schedulable resource.
+
+{% endcapture %}
+
+{% capture steps %}
+
+## API
+
+Huge pages can be consumed via container level resource requirements using the
+resource name `hugepages-{size}`, where size is the most compact binary notation
+using integer values supported on a particular node.  For example, if a node
+supports 2048kB page sizes, it will expose a schedulable resource
+`hugepages-2Mi`.  Unlike CPU or memory, huge pages do not support overcommit.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  generateName: hugepages-volume-
+spec:
+  containers:
+  - image: fedora:latest
+    command:
+    - sleep
+    - inf
+    name: example
+    volumeMounts:
+    - mountPath: /hugepages
+      name: hugepage
+    resources:
+      limits:
+        hugepages-2Mi: 100Mi
+  volumes:
+  - name: hugepage
+    emptyDir:
+      medium: HugePages
+```
+
+- Huge page requests must equal the limits.
```

Comment:
nit: might add "This is the default if limits are specified but requests are not."